scripts/get-gee-build-environment.sh

```diff
@@ -0,0 +1,421 @@
+#!/usr/bin/env bash
+
+# Copyright 2017 OpenGEE Contributors
```

Comment:
I think we're developing under Apache 2.0, so you may want to add this to the copyright notice.

Reply:
This is the heading block that should be used:

```
#
# Copyright 2017 Open GEE Contributors
#
# Licensed under the Apache License, Version 2.0 (the "License");
# you may not use this file except in compliance with the License.
# You may obtain a copy of the License at
#
#      http://www.apache.org/licenses/LICENSE-2.0
#
# Unless required by applicable law or agreed to in writing, software
# distributed under the License is distributed on an "AS IS" BASIS,
# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
# See the License for the specific language governing permissions and
# limitations under the License.
#
```